src/pages/home/report/ReportActionsView.js

```diff
@@ -114,6 +122,10 @@ class ReportActionsView extends React.Component {
         this.recordTimeToMeasureItemLayout = this.recordTimeToMeasureItemLayout.bind(this);
         this.scrollToBottomAndUpdateLastRead = this.scrollToBottomAndUpdateLastRead.bind(this);
         this.updateNewMarkerAndMarkReadOnce = _.once(this.updateNewMarkerAndMarkRead.bind(this));
+        this.scrollToReportActionID = this.scrollToReportActionID.bind(this);
+        this.recordReportActionIDRendered = this.recordReportActionIDRendered.bind(this);
+        this.recordMeasurementDone = this.recordMeasurementDone.bind(this);
+        this.checkScrollToReportAction = this.checkScrollToReportAction.bind(this);
```

Comment:
**question:** Is it possible to move some of this logic to `ReportActionsList` itself to maybe keep this component a bit lighter? It's one of our more complex ones and can be hard to make sense of everything it is responsible for. I feel like maybe some of the things we are adding could be the responsibility of the the "list" instead of the "view".

Reply:
We can always look at it in a follow up. My desire to keep this component simple shouldn't necessarily stand in the way of implementing this feature.